app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -22,12 +22,12 @@ def linkedin
     if @user && @user.persisted?
       sign_in @user, event: :authentication #this will throw if @user is not activated
       set_flash_message(:notice, :success, kind: "LinkedIn") if is_navigational_format?
-      redirect_to employer_home_path
+      redirect_to edit_employer_path(current_user.employer)
```

Comment:
With this, when a user is on the commitments page and logs in, they are now sent to the "edit employer" page rather than back to the commitments page. @patrickvinograd thoughts on if they should go back to where they started or if this is better?  This is a product decision that you and I are just going to make, so curious if you feel strongly one way or another.